src/registrar/context_processors.py

```diff
@@ -0,0 +1,6 @@
+from django.conf import settings
+
+
+def language_code(request):
+    """Add LANGUAGE_CODE to the template context."""
```

Comment:
I'm not certain why we need to include `settings.LANGUAGE_CODE` in the context. Would you mind expanding the docstring to explain the purpose of this?